.circleci/scripts/deps-install.sh

```diff
@@ -5,4 +5,10 @@ set -e
 set -u
 set -o pipefail
 
-yarn --frozen-lockfile --ignore-scripts --har
+cd ./packages
+
+# ref: https://github.com/koalaman/shellcheck/wiki/SC2044
+find . -mindepth 1 -maxdepth 1 -type d -exec sh -c '
+    cd "$1"
+    yarn --frozen-lockfile --ignore-scripts --har
```

Comment:
Ideally we'd install dependencies _once_ at the top-level. That would eliminate the need for these two loops.

Reply:
Hehe, yeah, it was only installing once even with multiple packages in my follow-up PR 😅 

f05e39f